packages/backend/src/services/UserAttributesService/UserAttributesService.ts

```diff
@@ -0,0 +1,86 @@
+import { subject } from '@casl/ability';
+import {
+    CreateOrgAttribute,
+    ForbiddenError,
+    SessionUser,
+    UserAttribute,
+} from '@lightdash/common';
+import { UserAttributesModel } from '../../models/UserAttributesModel';
+
+type Dependencies = {
+    userAttributesModel: UserAttributesModel;
+};
+
+export class UserAttributesService {
+    private readonly userAttributesModel: UserAttributesModel;
+
+    constructor(dependencies: Dependencies) {
+        this.userAttributesModel = dependencies.userAttributesModel;
+    }
+
+    async getAll(user: SessionUser): Promise<UserAttribute[]> {
+        if (user.ability.cannot('manage', 'Organization')) {
```

Comment:
Shouldn't we check the organization? 

```typescript
user.ability.cannot('manage', subject('Organization', { organizationUuid }))
```

I know each user now has one organization but if we assume that then it couples this function to that business logic but it doesn't need to be. Makes it easier when we allow many orgs per user